frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -2041,8 +2041,9 @@ function ReaderFooter:onHoldFooter()
     if self.mode == self.mode_list.off then return end
     if self.settings.skim_widget_on_hold then
         self.ui:handleEvent(Event:new("ShowSkimtoDialog"))
+        return true
     end
-    return true
+    return false
```

Comment:
Probably better as "nothing". I don't exactly remember how this is handled on our side, but Lua has weird conventions where retruning nothing is slightly different from returning false, and also slightly different than returning nil ;).

Reply:
`return nil` is the same as returning nothing. ;-)

Reply:
There are apparently sneaky subtleties: http://lua-users.org/lists/lua-l/2011-09/msg00289.html

Reply:
Was pretty surprised, but there is some difference https://stackoverflow.com/a/18526055

Reply:
Fair, not 100% the same thing, but not different different like false. Eh, never mind. But just get rid of the line. :-P